src/utils/makeReactNativeConfig.js

```diff
@@ -197,6 +213,10 @@ const getDefaultConfig = (
     mainFields: ['react-native', 'browser', 'main'],
     extensions: [`.${platform}.js`, '.native.js', '.js'],
   },
+  /**
+   * Set target to webworker as it's closer to RN environment than `web`.
+   */
+  target: 'webworker',
```

Comment:
Different files with templates 😛 I think plugins and loader might behave differently. But it's possible that plugins and loaders get templates from `compiler` / `compilation` so in theory everything should be fine.